src/main/java/pro/sky/homework9/controller/EmployeeController.java

```diff
@@ -3,15 +3,18 @@
 import org.springframework.web.bind.annotation.*;
 import pro.sky.homework9.model.Employee;
 import pro.sky.homework9.service.EmployeeService;
+import pro.sky.homework9.service.EmployeeServiceImpl;
 
 import java.util.Collection;
+import java.util.List;
+import java.util.Map;
 
 @RestController
 @RequestMapping("/employee")
 public class EmployeeController {
     private final EmployeeService employeeService;
 
-    public EmployeeController(EmployeeService employeeService) {
+    public EmployeeController(EmployeeServiceImpl employeeService) {
```

Comment:
```suggestion
    public EmployeeController(EmployeeService employeeService) {
```
инжектить надо всегда по интерфейсу, а не реализации